lib/core/src/putUtil.cpp

```diff
@@ -674,19 +660,11 @@ initCondForPut( rcComm_t *conn, rodsEnv *myRodsEnv, rodsArguments_t *rodsArgs,
         return USER_INPUT_OPTION_ERR;
     }
 
-    /* Not needed - dataObjOprInp->createMode = 0700; */
-    /* mmap in rbudp needs O_RDWR */
-    dataObjOprInp->openFlags = O_RDWR;
```

Comment:
Is this line specifically for RBUDP?

Reply:
Removed it based on the comment: `/* mmap in rbudp needs O_RDWR */`.

Reply:
Is openFlags referenced anywhere else in putUtil.cpp? How about rsDataObjPut.cpp?

Reply:
I've taken a brief loot at it. This seems to be the only spot that sets `openFlags` in `putUtil.cpp`. `dataObjPut.h` has documentation that states for `rcDataObjPut(...)` that `openFlags` "should be set to O_WRONLY".

`rsDataObjPut.cpp` does use `openFlags`, it sets it to `O_RDWR` after a `...PostProc...(...)`. The `DataObjInp` is also passed around, so it may be used by some other deeper functions.

We can revert this change so the `openFlags` has a value set, and maybe open an issue to check that functions mentioning `openFlags` have up-to-date documentation.

Reply:
Let's assume that line is for RBUDP and continue moving forward.

Open an issue stating that we removed this and that we need to verify topology tests  (launched from a consumer) pass.

Link this comment to the issue so we can find it later. If topology tests pass, then we can assume it's safe to just close the issue.

Reply:
You do not need to run the topology tests for this PR. We can do that later.

Reply:
Issue created: #8336